packages/logging_cloudwatch/amplify_logging_cloudwatch/example/lib/homepage.dart

```diff
@@ -122,15 +120,15 @@ class _MyHomePageState extends State<MyHomePage> {
   }
 
   void _enableLogging() {
-    _plugin.enable();
+    Amplify.Logging.enable();
   }
 
   void _disableLogging() {
-    _plugin.disable();
+    Amplify.Logging.enable();
```

Comment:
is this supposed to be `Amplify.Logging.enable()` for the `_disableLogging()` call?

Reply:
ops, I will fix it